packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/TxDetailModal/Detail/AdvancedTxDetails/IODetails/TransactionDetailsModal.tsx

```
@@ -0,0 +1,118 @@
+                                    </Paragraph>
+                                    {'\n'}
+                                    <Translation id="TR_TO" />:{'\n'}
+                                    <Paragraph isMonospaced>
+                                        {token.to || <Translation id="TR_UNKNOWN_ADDRESS" />}
+                                        {'\n'}
+                                        {formatAmount(token.amount, token.decimals)}{' '}
+                                        {getDisplaySymbol(token.symbol ?? token.name ?? '')}
+                                    </Paragraph>
+                                </>
+                            ))}
+                        </>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)